src/governance/governance.h

```diff
@@ -293,8 +293,9 @@ class CGovernanceManager : public GovernanceStore
 
     void ProcessMessage(CNode& peer, PeerManager& peerman, CConnman& connman, std::string_view msg_type, CDataStream& vRecv);
 
-    std::optional<CSuperblock> CreateSuperblockCandidate(int nHeight) const;
-    void CreateGovernanceTrigger(const CSuperblock& sb, CConnman& connman);
+    const std::optional<CSuperblock> CreateSuperblockCandidate(int nHeight) const;
+    const std::optional<CGovernanceObject> CreateGovernanceTrigger(const std::optional<CSuperblock>& sb_opt, CConnman& connman);
```

Comment:
what is your goal marking these returned values as const?

Reply:
I think probably if anything you may want `std::optional<const T>` but probably not `const std::optional<T>`

Reply:
oops 🙈 fixed